### PR TITLE
Provide a more useful error in the modal when there is a problem

### PR DESCRIPTION
**Pull request by jcoyne**
Screen Shot 2020-11-30 at 2 14 55 PM

Previously it just said "Network Error"

### CODE REVIEW

File: app/javascript/blacklight/modal.js [hunk 1]
```diff
@@ -101,12 +101,16 @@ Blacklight.modal.modalCloseSelector   = '[data-blacklight-modal~=close]';
...
 // network errors.
-Blacklight.modal.onFailure = function(data) {
-  var contents =  '<div class="modal-header">' +
-            '<div class="modal-title">Network Error</div>' +
-            '<button type="button" class="blacklight-modal-close close" data-dismiss="modal" aria-label="Close">' +
-            '  <span aria-hidden="true">&times;</span>' +
-            '</button>';
+Blacklight.modal.onFailure = function(jqXHR, textStatus, errorThrown) {
+  console.error('Server error:', this.url, jqXHR.status, errorThrown)
+  var contents =  `<div class="modal-header">
```
Comment (mejackreed) [inline app/javascript/blacklight/modal.js:106]: Just dropping in the support matrix for template literals. This change would not be parsable by IE11.

https://caniuse.com/?search=template%20literals

Comment (jcoyne) [reply app/javascript/blacklight/modal.js:106]: I thought we heard agreement on https://github.com/projectblacklight/blacklight/issues/2332 at blacklight conf.

Comment (mejackreed) [reply app/javascript/blacklight/modal.js:106]: Maybe we did? Honestly I don't remember.

**Comment by jcoyne** [comment]
superseded by #2407

**Pull request closed without merging**